### ISSUE TITLE
2019 oemof Spring Meeting

### ISSUE THREAD

**Issue by p-snft** [oemof/organisation#43]
It was suggested to hold the next oemof meeting in Oldenburg. According to the schedule, it should be a user meeting. I checked back with our administration and they would agree to host the event from Wednesday the 15th to Friday the 17th of May. This is the time, we have room easily available. If the suggested dates do not fit, we would have to look if it is still possible. (I have a tentative reservation for the room for the dates named above.)

**Comment by fwitte** [comment]
Thank you for your effort! The date looks good for me. How do you suggest, we get in contact with possible users (maybe a routine is already existing), @oemof-developer?

**Comment by ckaldemeyer** [comment]
I am fully packed in May and might show up spontaneously depending on how my work load evolves :/

**Comment by simnh** [comment]
I am interested and I do not have anything to do in May ;-)

**Comment by uvchik** [comment]
I will attend at least one of the two days. Thank you @p-snft for hosting the next meeting.

**Comment by simnh** [comment]
Ah shit, maybe I was a bit too enthusiastic. Unfortunately I can not attend, as some friends will marry at that time. Actually this might reduce the number of Flensburg (at least @gnn, I think)...Though Thursday might be possible

**Comment by SabineHaas** [comment]
For me the dates are not so convenient either as I will be at a project meeting on Monday after these dates, which is far too far from Berlin (....and even further from Oldenburg) and I guess I'll have lots of things to prepare before that.

**Comment by henhuy** [comment]
I will be in "parental leave" - so I cannot attend.

**Comment by p-snft** [comment]
@henhuy: That's a good reason. (For me, having a baby was tough but enjoyable. With routine, it becomes easier.)

**Comment by henhuy** [comment]
It's my second ;) But I appreciate your encouraging words!

**Comment by jakob-wo** [comment]
I am interested and the proposed date is convenient for me.

**Comment by birgits** [comment]
As @SabineHaas and possibly @gnn can/might not attend I could join as a feedinlib and windpowerlib representative.

**Comment by jnnr** [comment]
I will join. Looking forward to the meeting in May!

**Comment by jnnr** [comment]
Now, there is an announcement on the oemof website: 
https://oemof.org/2019/03/26/next-oemof-developer-and-user-meeting-in-may-2019/
Topics can be collected in advance here:
https://github.com/oemof/organisation/wiki/Meeting-May-2019

**Comment by p-snft** [comment]
@birgits It would make sense to have you around. (In particular, I would be interested in using windpowerlib for small wind turbines in urban areas. As this is probably not the standard use case, a chat on the applicability - and possible adjustments - of your models would be helpful.)

**Comment by birgits** [comment]
@p-snft I'd be happy to exchange ideas about that! It's already decided that I'll join for all three days.

**Comment by gnn** [comment]
I'd like to join, but I'm invited to the same wedding as @simnh, so I won't be able to attend on Friday. I'd be there on Wednesday and Thursday though.

**Comment by simnh** [comment]
I realised that for me there is no way to join this meeting. I know this might be a little unfortunate and also we talked about possibilities to arrange it. However there is no chance for me to attend without very big organisational and stressfull efforts which are   out of scope of my personal capabilities for the next months. I hope you have a good meeting. BTW. Maybe using webco infrastructure would generally be nice for such meetings.
